packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.json

```diff
@@ -74,5 +74,8 @@
   "telephony_line_phone_detail_link": "Fiche technique",
   "telephony_line_phone_order_validate": "Valider ma commande",
   "ovh_contracts_agree": "Je reconnais avoir pris connaissance des contrats ci-dessus.",
-  "telephony_line_phone_choice_fees_description": "{{feesText}} : {{feesPriceText}}"
+  "telephony_line_phone_choice_fees_description": "{{feesText}} : {{feesPriceText}}",
+  "telephony_line_phone_order_process_return_comment1": "Attention : Nous vous prions de bien vouloir retirer du matériel et de son emballage tout effet personnel, information ou élément qui n’aurait aucun lien avec le matériel retourné.",
+  "telephony_line_phone_order_process_return_comment2": "Dans le cas contraire, OVH tentera de prendre attache avec le client par courriel pour restituer les biens n’appartenant pas à OVH.",
```

Comment:
```suggestion
  "telephony_line_phone_order_process_return_comment2": "Dans le cas contraire, OVHcloud tentera de prendre attache avec le client par courriel pour restituer les biens n’appartenant pas à OVHcloud.",
```
maybe

Reply:
Must be validated first because the website isn't mentioning it yet (https://www.ovhtelecom.fr/)

Reply:
I need to ask to the person which makes this demand of change.